pandas/io/stata.py

```diff
@@ -1794,18 +1773,16 @@ def _do_convert_categoricals(
                     repeats = list(vc.index[vc > 1])
                     repeats = "-" * 80 + "\n" + "\n".join(repeats)
                     # GH 25772
-                    msg = """
-Value labels for column {col} are not unique. These cannot be converted to
-pandas categoricals.
-
-Either read the file with `convert_categoricals` set to False or use the
-low level interface in `StataReader` to separately read the values and the
-value_labels.
-
-The repeated labels are:
-{repeats}
-"""
-                    raise ValueError(msg.format(col=col, repeats=repeats))
+                    msg = (
+                        f"\nValue labels for column {col} are not unique. These "
+                        "cannot be converted to\npandas categoricals.\n\n"
+                        "Either read the file with `convert_categoricals` set "
+                        "to False or use the\nlow level interface in "
+                        "`StataReader` to separately read the values and the"
+                        "\nvalue_labels.\n\nThe repeated labels are:\n"
+                        f"{repeats}\n"
+                    )
```

Comment:
Some of these newlines can be removed.  Only keep the ones that separate paragraphs

Reply:
same above